src/core/Akka/Actor/Props.cs

```diff
@@ -689,92 +669,20 @@ public FactoryConsumer(Func<TActor> factory)
                 _factory = factory;
             }
 
-            public ActorBase Produce()
+            public ActorBase Produce(Props props)
             {
                 return _factory.Invoke();
```

Comment:
maybe insert here a safety type check with `props.Type` ?

Reply:
`FactoryConsumer` was not used, I reactivated it for the Producer by Custom-Functon